Build broken with undefined zerolog.LevelWidth

With more recent versions of the zerolog package, LevelWidth has been removed, and so this no longer builds:

```
$ go build
# github.com/danielgtaylor/openapi-cli-generator/cli
../../../go/src/github.com/danielgtaylor/openapi-cli-generator/cli/log.go:60:6: undefined: zerolog.LevelWidth
../../../go/src/github.com/danielgtaylor/openapi-cli-generator/cli/log.go:61:18: undefined: zerolog.LevelWidth
../../../go/src/github.com/danielgtaylor/openapi-cli-generator/cli/log.go:64:21: undefined: zerolog.LevelWidth
```
More specifically, this was broken by the recent rewriting of the [zerolog.ConsoleWriter interface](https://github.com/rs/zerolog/commit/96f91bb4f5dae9f5026ed08500dd8692f9cdf67b#diff-b0d3ae5e607395dec491b715aea0e01b).

--- a/cli/log.go
+++ b/cli/log.go
@@ -57,13 +57,6 @@ func (w ConsoleWriter) Write(p []byte) (n int, err error) {
 			lvlColor = levelColor(l)
 		}
 		level = strings.ToUpper(l)
-		if zerolog.LevelWidth > 0 {
-			if padding := zerolog.LevelWidth - len(level); padding > 0 {
-				level += strings.Repeat(" ", padding)
-			} else {
-				level = level[0:zerolog.LevelWidth]
-			}
-		}
 	}
 	fmt.Fprintf(buf, "%s %s %s",
 		colorize(level, lvlColor, !w.NoColor),

